### PR TITLE
Added freq display font selection.

### PR DESCRIPTION
**Pull request by sp9bsl**

### CODE REVIEW

File: mchf-eclipse/drivers/ui/menu/ui_menu_internal.h [hunk 1]
```diff
@@ -92,6 +92,7 @@ enum
     MENU_METER_COLOUR_DOWN,
     MENU_DBM_DISPLAY,
     MENU_DBM_CALIBRATE,
+	MENU_FREQ_FONT,
```
Comment (UR7FM) [inline mchf-eclipse/drivers/ui/menu/ui_menu_internal.h:95]: How to change  freq font? Im display menu I didnt see that.

Comment (sp9bsl) [reply mchf-eclipse/drivers/ui/menu/ui_menu_internal.h:95]: Which version of CPU do you have? This is available for 1024kB+ version.

Comment (UR7FM) [reply mchf-eclipse/drivers/ui/menu/ui_menu_internal.h:95]: 512kB :((

Comment (sp9bsl) [reply mchf-eclipse/drivers/ui/menu/ui_menu_internal.h:95]: Fonts needs memory. We had to limit unnecessary things (from HAM point of view) to keep support for older machines, sorry.

**Pull request merged**